cleo/descriptors/text_descriptor.py

```diff
@@ -69,27 +68,25 @@ def _describe_option(self, option: Option, **options: Any) -> None:
             "total_width", self._calculate_total_width_for_options([option])
         )
 
-        synopsis = "{}{}".format(
-            f"-{option.shortcut}, " if option.shortcut else "    ",
-            f"--{option.name}{value}",
+        synopsis = (
+            f'{f"-{option.shortcut}, " if option.shortcut else "    "}'
+            f'{f"--{option.name}{value}"}'
```

Comment:
double f-strings here 😄
```suggestion
            f"--{option.name}{value}"
```